src/beam.py

```diff
@@ -0,0 +1,25 @@
+from numpy.random import rand
+
+class Beam:
+    def __init__(self, start_angle, end_angle, length=1, color=[rand(), rand(), rand()]):
+        self.start_angle = start_angle
+        self.end_angle = end_angle
```

Comment:
Can we make the color `[0, 0.5, 0]` or some other consistent color?

Reply:
That works for me